Tools/gsc/finalize_manifests.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import re
+import string
+import argparse
+
+def is_ascii(chars):
+    return all(ord(c) < 128 for c in chars)
+
+def generate_trusted_files(root_dir):
+    # Exclude directories from list of trusted files
+    exclude_dirs = ['boot', 'dev', 'etc/rc', 'proc', 'sys', 'var']
```

Comment:
@vahldiek Although we discussed this today, adding comment in this PR for reference. In generate_trusted_files, we can add a check, to only include regular files, and not other file types such as sockets.